package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "customize-cra",
-  "version": "0.2.12",
+  "name": "@zhigang1992/customize-cra",
```

Comment:
Please don't edit the package.json, we'll do that when we release a new version

Reply:
sry... that wasn't mean to be included.